File: env.py
#!/bin/env python
import os, sys, cx_Oracle

print("Content-type: text/html\n")

# Get python location
stream = os.popen('which python')
output = stream.read()

print("python %s<br/>" % (sys.version))
print("%s<br/>" % output)
print("cx_Oracle %s<br/>" % cx_Oracle.version)
for k, v in sorted(os.environ.items()) :
    print("%s = %s<br/>" % (k, v))

File: env.pl
#!/bin/env perl
print "Content-type: text/html\n\n";

printf("%s<br/>", `perl -V`);
printf("%s<br/>", `which perl`);

foreach (sort keys %ENV) {
    printf("%s = %s<br>\n", $_, $ENV{$_});
}
